service/src/main/java/com/oasis/service/LoginService.java

```diff
@@ -0,0 +1,26 @@
+package com.oasis.service;
+
+import com.oasis.exception.UserNotFoundException;
+import com.oasis.model.EmployeeModel;
+import com.oasis.repository.LoginRepository;
+import com.oasis.webmodel.response.LoginResponseModelOK;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LoginService {
```

Comment:
please separate the interface and the implementation
LoginServiceApi & LoginServiceImpl
also separate the package for Api & Impl